docs/apis/batch/dataset_transformations.md

```diff
@@ -1010,7 +1013,10 @@ val output = input.aggregate(SUM, 0).and(MIN, 2)
 <div data-lang="python" markdown="1">
 
 ~~~python
-Not supported.
+from flink.functions.Aggregation import Sum, Min
+
+input = # [...]
+output = input.aggregate(Sum, 0).agg_and(Min, 2)
```

Comment:
documentation wasn't updated for the rename to `and_agg`. The same applies to the `python.md`